tests/lambda_endpoints/test_batch_starter.py

```diff
@@ -0,0 +1,258 @@
+import csv
+import datetime
+import json
+import zoneinfo
+
+import pytest
+
+from sds_data_manager.lambda_images.instruments.batch_starter import (
+    all_dependency_present,
+    get_filename_from_event,
+    get_process_details,
+    prepare_data,
+    query_dependencies,
+    query_dependents,
+    remove_ingested,
+)
+
+
+@pytest.fixture(scope="session")
+def mock_event():
+    """Example of the type of event that will be passed to
+    the instrument lambda (in our case batch_starter.py).
+    """
+    with open("../test-data/codicehi_event.json") as file:
+        event = json.load(file)
+    return event
+
+
+@pytest.fixture()
+def setup_database(postgresql):
```

Comment:
```suggestion
def database(postgresql):
```
Just call it `database`? You are setting up here, but the fixture is what you are using elsewhere.